src/components/atoms/Select/Select.tsx

```diff
@@ -120,8 +124,8 @@ export const Select: React.FC<SelectProps> = ({
       <div className='selectWrapper'>
         <select
           aria-labelledby={name}
-          value={valueState}
-          onChange={onValueChange}
+          {...!register ? { value: valueState, onChange: onValueChange } : null}
```

Comment:
Please move the ternary operation out of the destructuring statement by storing it's value in a variable and destructuring that. 
```
const myValue = !register ? { value: valueState, onChange: onValueChange } : null
//...
{...myValue}
```
Even better: use the `if` block in line 54 😉

Reply:
alright, i'll do that. thanks.